thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionPropertiesUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.thirdeye.anomaly.detection.lib;
+
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.dto.FunctionAutoTuneConfigDTO;
+import java.util.Map;
+import java.util.Properties;
+
+
+/**
+ * This utility class is used to apply tuning configuration to function properties, and convert properties to String
+ */
+public class FunctionPropertiesUtils {
```

Comment:
Any other codes are using this class/method. These codes looks very should and is there existing code that can do these; seems to me these are pretty common tasks.